tests/dedicatedvstests/l7_dedicated_graph_test.go

```diff
@@ -182,6 +182,41 @@ func SetupDomain() {
 	mcache.CloudKeyCache.AviCacheAdd("Default-Cloud", cloudObj)
 }
 
+func SetUpIngressForCacheSyncCheckMultiPaths(t *testing.T, tlsIngress, withSecret bool, fqdns []string, paths []string, modelNames ...string) {
```

Comment:
Can this function not be added in a lib file ?

Reply:
These functions have been defined for each package in _test.go files and using global variables shared with other functions in the same file. Also, variables defined in _test.go files are not accessible from normal .go files so adding these in a lib.go will also cause complexities. If required, this will have to taken up separately.